R/build-favicons.R

```diff
@@ -131,10 +131,13 @@ has_favicons <- function(pkg = ".") {
 
 find_logo <- function(path) {
   path_first_existing(
-    path(path, "logo.svg"),
-    path(path, "man", "figures", "logo.svg"),
-    path(path, "logo.png"),
-    path(path, "man", "figures", "logo.png")
+    path,
```

Comment:
I think it's a little simpler to understand if you leave the existing repetition of the `path` variable.  (i.e. just delete the other function and leave this one alone)

Reply:
You mean not amending this function at all? Then it doesn't work (I first only deleted the other one, that's how I know)

Reply:
Ooooh, you'll need to put a `c()` around the four paths.